### ISSUE TITLE
Fails to build contract because openzeppelin/contracts-ethereum-package asks Solidity v0.6.0

### ISSUE THREAD

**Issue by bafu** [thundercore/auction-template#1]
Fails to build contract because openzeppelin/contracts-ethereum-package asks Solidity v0.6.0.

**Steps to Reproduce**

1. Setup environment

        $ yarn install

2. Build contract

        $ npm run build

**Erro Logs**

[2021-04-19T04_09_22_988Z-debug.log.txt](https://github.com/thundercore/auction-template/files/6333210/2021-04-19T04_09_22_988Z-debug.log.txt)

 ```
 bafu@bafu-XPS-13-7390  ~/codes/auction-template   implement-auction  npm run build

> auction-template@1.0.0 build /home/bafu/codes/auction-template
> truffle compile


Compiling your contracts...
===========================
> Compiling ./contracts/Auction.sol
> Compiling ./contracts/AuctionFactory.sol
> Compiling ./contracts/interfaces/IAuction.sol
> Compiling ./contracts/interfaces/IAuctionFactory.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/Initializable.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/access/Ownable.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/math/SafeMath.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/token/ERC20/IERC20.sol

@openzeppelin/contracts-ethereum-package/contracts/access/Ownable.sol:1:1: ParserError: Source file requires different compiler version (current compiler is 0.5.17+commit.d19bba13.Emscripten.clang - note that nightly builds are considered to be strictly less than the released version
pragma solidity ^0.6.0;
^---------------------^
,@openzeppelin/contracts-ethereum-package/contracts/math/SafeMath.sol:1:1: ParserError: Source file requires different compiler version (current compiler is 0.5.17+commit.d19bba13.Emscripten.clang - note that nightly builds are considered to be strictly less than the released version
pragma solidity ^0.6.0;
^---------------------^
,@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/IERC20.sol:1:1: ParserError: Source file requires different compiler version (current compiler is 0.5.17+commit.d19bba13.Emscripten.clang - note that nightly builds are considered to be strictly less than the released version
pragma solidity ^0.6.0;
^---------------------^

Error: Truffle is currently using solc 0.5.17, but one or more of your contracts specify "pragma solidity ^0.6.0".
Please update your truffle config or pragma statement(s).
(See https://trufflesuite.com/docs/truffle/reference/configuration#compiler-configuration for information on
configuring Truffle to use a specific solc compiler version.)

Compilation failed. See above.
Truffle v5.3.1 (core: 5.3.1)
Node v14.16.1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! auction-template@1.0.0 build: `truffle compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the auction-template@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/bafu/.npm/_logs/2021-04-19T04_09_22_988Z-debug.log
```

**Environment**

```
$ truffle version
Truffle v5.3.1 (core: 5.3.1)
Solidity - 0.5.17 (solc-js)
Node v14.16.1
Web3.js v1.3.5
```

**Comment by o-p** [comment]
Should be fixed after #2 merged.

:smile:

**Comment by bafu** [comment]
Confirmed that this issue is fixed in commit `64bef0a` (current `master`)

```
$ yarn build
yarn run v1.22.5
$ truffle compile

Compiling your contracts...
===========================
> Compiling ./contracts/Auction.sol
> Compiling ./contracts/AuctionFactory.sol
> Compiling ./contracts/interfaces/IAuction.sol
> Compiling ./contracts/interfaces/IAuctionFactory.sol
> Compiling ./contracts/mocks/ERC20Token.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/GSN/Context.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/access/Roles.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/access/roles/MinterRole.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/access/roles/PauserRole.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/lifecycle/Pausable.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/math/SafeMath.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/ownership/Ownable.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/token/ERC20/ERC20.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/token/ERC20/ERC20Detailed.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/token/ERC20/ERC20Mintable.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/token/ERC20/ERC20Pausable.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/token/ERC20/IERC20.sol
> Compiling @openzeppelin/contracts-ethereum-package/contracts/token/ERC20/StandaloneERC20.sol
> Compiling @openzeppelin/upgrades/contracts/Initializable.sol
> Artifacts written to /home/bafu/codes/auction-template/build/contracts
> Compiled successfully using:
   - solc: 0.5.17+commit.d19bba13.Emscripten.clang

Done in 7.24s.
```
